.github/workflows/ci.yaml

```diff
@@ -16,3 +16,21 @@ jobs:
         name: pre-commit-hooks
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
     - run: nix-build
+  tests-flakes:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v13
+      with:
+          install_url: https://nixos-nix-install-tests.cachix.org/serve/lb41az54kzk6j12p81br4bczary7m145/install
+          install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+    - uses: cachix/cachix-action@v10
+      with:
+        name: pre-commit-hooks
+        signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+    - run: nix flake check
```

Comment:
I can also recommend `nix flake show` to validate the flake output schema: https://github.com/Mic92/sops-nix/blob/4f384662a85804fa2bc1bc1f99e70bb468e76f88/.github/workflows/test-flakes.yml#L29

Reply:
Thanks!